CalibCalorimetry/HcalPlugins/python/Hcal_Conditions_forGlobalTag_cff.py

```diff
@@ -161,3 +162,34 @@
                              useHEUpgrade = cms.bool(True),
                              useHFUpgrade = cms.bool(True)
 )
+
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+phase2_hgcal.toModify( es_hardcode,
+                       toGet = cms.untracked.vstring(
+                                         'GainWidths',
+                                         'MCParams',
+                                         'RecoParams',
+                                         'RespCorrs',
+                                         'QIEData',
+                                         'QIETypes',
+                                         'Gains',
+                                         'Pedestals',
+                                         'PedestalWidths',
+                                         'ChannelQuality',
+                                         'ZSThresholds',
+                                         'TimeCorrs',
+                                         'LUTCorrs',
+                                         'LutMetadata',
+                                         'L1TriggerObjects',
+                                         'PFCorrs',
+                                         'FrontEndMap',
+                                         'CovarianceMatrices',
+                                         'SiPMParameters',
+                                         'SiPMCharacteristics',
+                                         'TPChannelParameters',
+                                         'TPParameters',
+                                         'FlagHFDigiTimeParams'
+                                         ),
```

Comment:
@mmusich it's possible, but phase1 is the priority right now and I'm not sure how much spare personpower @abdoulline has to be propagating changes to Phase2 also (esp. given geometry/topology variations that would necessitate different tags for different phase2 workflows)